infrastructure/testdriver/actions/pause.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>TestDriver actions: event order</title>
```

Comment:
Mind updating the title?

Reply:
Sure. I will also need to skip the test for Chrome as we don't support it yet in ChromeDriver.